lib/codegen/ProofEvent.cpp

```diff
@@ -174,7 +174,7 @@ proof_event::event_prelude(
 llvm::BasicBlock *proof_event::hook_event_pre(
     std::string const &name, llvm::BasicBlock *current_block,
     std::string const &location_stack) {
-  if (!proof_hint_instrumentation) {
```

Comment:
I think it would be better if setting `--slow` also set the regular flag rather than having to make the code check both flags everywhere.

Reply:
Good point, I pushed a commit to do that.